### DISCUSSION TITLE
1.6.0 미만의 버전에서 Character culling을 사용하고 싶습니다

### DISCUSSION THREAD

**Discussion by qeasd11** [naverz/zepeto-studio-kor#1240] [5. Q&A]
카메라와 플레이어의 콜라이더가 부딪히면 캐릭터를 숨김처리하려고 하는데 카메라에 박스 콜라이더를 추가하니 빠른 속도로 카메라가 앞뒤로 움직입니다

Character culling을 구현할 좋은 방법 없을까요?ㅠ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. https://github.com/naverz/zepeto-studio-kor/discussions/1221#discussioncomment-3523637 답변과 마찬가지로 구버전 카메라에 지원하는 것은 불가합니다.
1.6.0 버전의 개선된 카메라에서 설정값을 조절하며 원하는 효과를 구현하시는 방법을 추천 드립니다.
